src/main/webapp/js/webrtc_adaptor.js

```diff
@@ -773,6 +773,32 @@ export class WebRTCAdaptor {
         this.webSocketAdaptor.send(JSON.stringify(jsCmd));
     }
 
+    /**
+     * Called to get the list of video track assignments. AMS responds with the videoTrackAssignmentList message.
+     * Parameters:
+     * @param {string} streamId: unique id for the stream that you want to get info about
+     */
+    requestVideoTrackAssignments(streamId) {
```

Comment:
Please make the name and command similar. I mean use `getVideoTrackAssignments` as you do `getBroadcastObject`